client/src/App.jsx

```diff
@@ -34,7 +35,14 @@ const theme = createMuiTheme({
   }
 });
 
-const App = () => (
+const contextDefaultValue = {
```

Comment:
initialize the context when the app starts to control the starting of the tutorial on the covid page